captcha/image.py

```diff
@@ -157,19 +160,20 @@ def create_noise_dots(image, color, width=3, number=30):
             number -= 1
         return image
 
-    def create_captcha_image(self, chars, color, background):
+    def create_captcha_image(self, chars: str, color: tuple, background: tuple, rotate: bool = True):
```

Comment:
this `tuple` type hint is not compatible with 3.8.

Reply:
I use Python 3.8 in almost all of my projects and this hint works fine. The project using this library with my modifications is no exception.

Reply:
It is working if not parametrized, but using `tuple[...]` is [3.9+](https://docs.python.org/3/library/typing.html#typing.Tuple).

(It is usually recommended to parametrize generic types, if you don't know the structure of the `color` argument, you can use `typing.Tuple[T, ...]`)

Reply:
why dont yall just `from __future__  import annotations`